### DISCUSSION TITLE
User Table Editing

### DISCUSSION THREAD

**Discussion by BTProjects** [Budibase/budibase#13890] [Help] [answered]
If I want to associate a user to say an agency, how can I do this?  It looks like the user table cannot be edited to add columns.

**Comment by mikesealey** [comment, marked as answer]
The user table cannot be edited directly, but it can have _relationships_ with other tables, which is likely how you should tackle this.

Create an `agencies` table and populate it with the relevant data, and then add a `relationship` column to join the `Users` table
![image](https://github.com/Budibase/budibase/assets/39269380/cc368309-6b39-4fc5-bdb7-14b6f5a3ac33)



Now in the agencies table, adding a user to that row will populate the agency in the `App users` table.
![image](https://github.com/Budibase/budibase/assets/39269380/5c27e683-896a-4321-a9ed-2c9793044b82)

**Comment by BTProjects** [comment]
Thanks Mike.  I was playing around with this concept but noted that if you do not correctly setup the relationship with the Users table (it was just user error), there was no way to remove the relationship related column without starting over with a new project, is this correct?

Based on this use case, let’s say I want my NASA agency to create clients (id, name, date of birth, etc) how does one accomplish this so they only create clients for NASA and do not have an option to expose or add to other client agencies?



Aaron Lind

W: 531-355-1484   M: 402.676.3064
[National_Logo.png]<https://www.boystown.org/>
boystown.org<https://www.boystown.org/>
14100 Crawford Street • Boys Town, NE 68010
***@***.***<https://www.linkedin.com/company/boys-town>   ***@***.*** <https://www.twitter.com/boystown>    ***@***.*** <https://www.facebook.com/boystown>
This message is privileged and confidential and is intended only for the use of the addressee. Unauthorized use of this message is strictly prohibited. If you received this transmission in error, please immediately contact the sender and destroy the message. Thank you.
